### PR TITLE
Product Service Updated

### PR DESCRIPTION
**Pull request by umeshtiwari123**

### CODE REVIEW

**Review by karthikkanse — COMMENTED** [review]
at line 34 
use the method of Optional class inside if condition

File: src/main/java/com/ty/fabrico/fabrico_springboot/service/ProductService.java [hunk 1]
```diff
@@ -1,15 +1,81 @@
...
+	
+	public ResponseEntity<ResponseStructure<Product>> saveProduct(Product product)
+	{
+		ResponseStructure<Product> responseStructure=new ResponseStructure<Product>();
+		ResponseEntity<ResponseStructure<Product>> responseEntity;
+
+		responseStructure.setStatus(HttpStatus.CREATED.value());
+		responseStructure.setMessage("Product Saved");
+		responseStructure.setData(productDao.saveProduct(product));
+		return responseEntity=new ResponseEntity<ResponseStructure<Product>>(responseStructure,HttpStatus.NOT_FOUND);
```
Comment (Manojkulli) [inline src/main/java/com/ty/fabrico/fabrico_springboot/service/ProductService.java:26]: Change NOT_FOUND to CREATED

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,7 +1,7 @@
 
 spring.datasource.url= jdbc:mysql://localhost:3306/Fabricoo_db?createDatabaseIfNotExist=true
 spring.datasource.username=root
-spring.datasource.password=root
+spring.datasource.password=TestYantra@2022
```
Comment (Manojkulli) [inline src/main/resources/application.properties:4]: make password as root

File: src/main/java/com/ty/fabrico/fabrico_springboot/service/ProductService.java [hunk 3]
```diff
@@ -1,15 +1,81 @@
...
+			{
+			product.setPId(productId);
+			responseStructure.setStatus(HttpStatus.OK.value());
+			responseStructure.setMessage("Updated");
+			responseStructure.setData(productDao.updateProduct(product));
+			return responseEntity;
+			
+			}
+
+		else throw new NoSuchIdFoundException("No Product Id is found for Updation");
```
Comment (Manojkulli) [inline src/main/java/com/ty/fabrico/fabrico_springboot/service/ProductService.java:78]: write message as  No Id Found Unable To Update

**Review by Manojkulli — COMMENTED** [review]
change the code

**Review by Manojkulli — APPROVED** [review]

**Pull request merged**